### PR TITLE
Add command response

### PR DESCRIPTION
**Pull request by lnx85**
As discussed [here](https://github.com/DeebotUniverse/client.py/discussions/507), this PR implements Command response.

## Summary by CodeRabbit

- **New Features**
  - Enhanced command execution to provide detailed response data.

- **Bug Fixes**
  - Improved handling of lifespan and position data in command responses.
  - Updated test assertions to correctly verify command execution results.

- **Documentation**
  - Removed outdated import statements from README.

- **Tests**
  - Added new test functions to validate lifespan and position command responses.
  - Updated existing tests to accommodate new response handling mechanisms.

### CODE REVIEW

**Comment by lnx85** [comment]
@coderabbitai review

**Review by edenhaus — CHANGES_REQUESTED** [review]

File: deebot_client/command.py [hunk 1]
```diff
@@ -73,16 +96,26 @@ async def execute(
         authenticator: Authenticator,
         device_info: ApiDeviceInfo,
         event_bus: EventBus,
-    ) -> bool:
+        response_type: CommandResponseType = CommandResponseType.STATUS_ONLY,
+    ) -> bool | Any:
```
[the remark this answers is not in the document]
Comment (lnx85) [reply deebot_client/command.py:100]: Is changing return type from bool to tuple[bool, dict[str, Any] backward compatible?

**Review by edenhaus — CHANGES_REQUESTED** [review]
Also you need to apply the following diff:
```diff
diff --git a/deebot_client/command.py b/deebot_client/command.py
index 971e82f..9f07e36 100644
--- a/deebot_client/command.py
+++ b/deebot_client/command.py
@@ -134,7 +134,7 @@ class Command(ABC):
         authenticator: Authenticator,
         device_info: ApiDeviceInfo,
         event_bus: EventBus,
-    ) -> CommandResult:
+    ) -> tuple[CommandResult, dict[str, Any]]:
         """Execute command."""
         try:
             response = await self._execute_api_request(authenticator, device_info)
@@ -143,21 +143,24 @@ class Command(ABC):
                 "Could not execute command %s: Timeout reached",
                 self.name,
             )
-            return CommandResult(HandlingState.ERROR)
+            return CommandResult(HandlingState.ERROR), {}
 
         result = self.__handle_response(event_bus, response)
         if result.state == HandlingState.ANALYSE:
             _LOGGER.debug(
                 "ANALYSE: Could not handle command: %s with %s", self.name, response
             )
-            return CommandResult(
-                HandlingState.ANALYSE_LOGGED,
-                result.args,
-                result.requested_commands,
+            return (
+                CommandResult(
+                    HandlingState.ANALYSE_LOGGED,
+                    result.args,
+                    result.requested_commands,
+                ),
+                response,
             )
         if result.state == HandlingState.ERROR:
             _LOGGER.warning("Could not parse %s: %s", self.name, response)
-        return result
+        return result, response
 
     async def _execute_api_request(
         self, authenticator: Authenticator, device_info: ApiDeviceInfo
diff --git a/deebot_client/commands/json/clean.py b/deebot_client/commands/json/clean.py
index c07c78b..469ac3f 100644
--- a/deebot_client/commands/json/clean.py
+++ b/deebot_client/commands/json/clean.py
@@ -32,7 +32,7 @@ class Clean(ExecuteCommand):
         authenticator: Authenticator,
         device_info: ApiDeviceInfo,
         event_bus: EventBus,
-    ) -> CommandResult:
+    ) -> tuple[CommandResult, dict[str, Any]]:
         """Execute command."""
         state = event_bus.get_last_event(StateEvent)
         if state and isinstance(self._args, dict):
```

**Comment by lnx85** [comment]
@edenhaus I'm having a problem with test_device.py but I can't find the reason. Any hint?

**Review by edenhaus — APPROVED** [review]
Thanks @lnx85 :+1:

**Pull request merged**